manifests/ulimit.pp

```diff
@@ -50,7 +51,7 @@
         "set \$nofile/value \"${redis::ulimit}\"",
       ],
       notify  => [
-        Exec['systemd-reload-redis'],
+        Class['systemd::systemctl::daemon_reload'],
```

Comment:
Should this notify also be hidden behind an if condition?

Reply:
OK, will add a condition.